localstripe/resources.py

```diff
@@ -3517,6 +3517,22 @@ def __init__(self, line_items=None, mode=None, success_url=None,
         self.subscription_data = subscription_data
         self.payment_intent_data = payment_intent_data
         self.payment_intent = pi.id if pi else None
+        self.status = "open"
 
     def _complete_session(self):
+        self.status = "complete"
         schedule_webhook(Event('checkout.session.completed', self))
+
+    def _expire_session(self):
+        self.status = "expired"
+        schedule_webhook(Event('checkout.session.expired', self))
+
+    @classmethod
+    def _api_expire(cls, id):
+        obj = Session._api_retrieve(id)
+        obj._expire_session()
+        # also cancels payment intent
+        PaymentIntent._api_cancel(obj.payment_intent)
```

Comment:
```suggestion
        # also cancels payment intent
        if obj.payment_intent:
            PaymentIntent._api_cancel(obj.payment_intent)
```
This can be none, for subscriptions